Core/Graphics/InternalInclude/Babylon/Graphics/FrameBuffer.h

```diff
@@ -20,7 +20,7 @@ namespace Babylon::Graphics
     class FrameBuffer final
     {
     public:
-        FrameBuffer(DeviceContext& context, bgfx::FrameBufferHandle handle, uint16_t width, uint16_t height, bool defaultBackBuffer, bool hasDepth, bool hasStencil);
+        FrameBuffer(DeviceContext& context, bgfx::FrameBufferHandle handle, uint16_t width, uint16_t height, bool defaultBackBuffer, bool hasDepth, bool hasStencil, int8_t depthStencilAttachmentIndex = -1);
```

Comment:
I think we can probably remove the hasDepth and hasStencil booleans. If depthStencilAttachmentIndex is not -1, then it has both depth and stencil, right?

Reply:
We already have this check on the native engine side.